.env.example

```diff
@@ -30,7 +30,7 @@ HAPI_RATE_HISTORY=https://api.coingecko.com/api/v3
 #webapp
 PORT=3000
 REACT_APP_TAG=localdev
-REACT_APP_NAME=$npm_package_name
+REACT_APP_NAME=0.0.1
 REACT_APP_TITLE=Eden on EOS Accounting Standard
 REACT_APP_LOGO=https://raw.githubusercontent.com/eoscostarica/design-assets/master/logos/eosCR/fullColor-vertiall-transparent-white.png
```

Comment:
are we using this en var?

Reply:
i dont know, but if i use `$npm_package_name` apear several warnings and on workflows appear like `eosreports`

Reply:
Ok, i deleted it and work fine, so i erase it?